CMakeLists.txt

```
@@ -45,12 +45,13 @@ target_include_directories(${PROJECT_NAME} INTERFACE
 )
 
 # Install Package Configuration
-install(TARGETS ${PROJECT_NAME} EXPORT ${PROJECT_NAME}_targets)
-install(EXPORT ${PROJECT_NAME}_targets
+string(TOLOWER ${PROJECT_NAME} PROJECT_NAME_LOWER)
+install(TARGETS ${PROJECT_NAME} EXPORT ${PROJECT_NAME_LOWER}_targets)
+install(EXPORT ${PROJECT_NAME_LOWER}_targets
     NAMESPACE ${PROJECT_NAME}::
-    FILE ${PROJECT_NAME}-config.cmake
+    FILE ${PROJECT_NAME_LOWER}-config.cmake

```

```suggestion
    FILE ${PROJECT_NAME_LOWER}-config.cmake
```suggestion
    FILE ${PROJECT_NAME_LOWER}Config.cmake

```
```